templates/manifest.json

```diff
@@ -219,7 +219,7 @@
         {
             "id": "python-crawlee-playwright",
             "name": "python-crawlee-playwright",
-            "label": "Start with Python Crawlee and Playwright",
+            "label": "Crawlee + Playwright + Chrome",
```

Comment:
but the template is using chromium now, right? 🙃

Reply:
Correct. Feel free to open an issue to straighten that up :slightly_smiling_face: